Taskfile.yaml

```diff
@@ -41,6 +43,18 @@ tasks:
       - golangci-lint run --fix
       - nilaway -fix -test=false -include-pkgs='github.com/opslevel/opslevel-jq-parser' ./...
 
+  parse:
+    desc: Locally parse config YAML files. Optionally pass in config with 'task parse -- myconfig.yaml'
+    dir: "{{.LOCAL_DIR}}"
+    cmds:
+      - '[[ -s  {{.LOCAL_CONFIG_YAML}} ]] || cp {{.ROOT_DIR}}/testdata/sample_config.yaml {{.LOCAL_CONFIG_YAML}}'
```

Comment:
`man bash`

>If the -s option is present, or if no arguments remain after option processing,
                 then commands are read from the standard input.

Do we want to use this?